### PR TITLE
Add task solution

### PR DESCRIPTION
**Pull request by andreinik1**
- [DEMO LINK] (https://andreinik1.github.io/layout_moyo-header/)
- [TEST REPORT LINK] (https://andreinik1.github.io/layout_moyo-header/report/html_report/)

- [x] Header height is set in 1 place (for the links)
- [x] Content is vertically centered (for any header height)
- [x] CSS is used to show all letters in Uppercase (don't type them in HTML)
- [x] Logo is an image wrapped with a link
- [x] CSS Variable is used for a blue color
- [x] Pseudo-element is used for a blue line below the active link
- [x] Code follows all the [Code Style Rules ❗️](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,6 +9,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header>
+      <div class="container">
+        <a href="./index.html"><img src="./images/logo.png" alt="MOYO Logo image"></a>
```
Comment (anastasiiavorobiova) [inline src/index.html:15]:
```suggestion
        <a href="./index.html">
          <img src="./images/logo.png" alt="MOYO Logo image">
        </a>
```

File: src/index.html [hunk 2]
```diff
@@ -9,6 +9,41 @@
...
+            <li class="nav__item">
+              <a href="./index.html" class="nav__link">Samsung</a>
+            </li>
+            <li class="nav__item">
+              <a href="./index.html" class="nav__link">Smartphones</a>
+            </li>
+            <li class="nav__item">
+              <a href="./index.html" data-qa="hover"
+                class="nav__link">Laptops & Computers
+              </a>
```
Comment (anastasiiavorobiova) [inline src/index.html:30]:
```suggestion
              <a 
                href="./index.html" 
                data-qa="hover"
                class="nav__link"
              >
                Laptops & Computers
              </a>
```

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,95 @@
...
+.container {
+  padding: 0 50px;
+
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.container img {
```
Comment (anastasiiavorobiova) [inline src/style.css:27]: Consider using class to style the image

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done!

**Comment by andreinik1** [comment]
- [DEMO LINK] (https://andreinik1.github.io/layout_moyo-header/)
- [TEST REPORT LINK] (https://andreinik1.github.io/layout_moyo-header/report/html_report/)

- [x] Header height is set in 1 place (for the links)
- [x] Content is vertically centered (for any header height)
- [x] CSS is used to show all letters in Uppercase (don't type them in HTML)
- [x] Logo is an image wrapped with a link
- [x] CSS Variable is used for a blue color
- [x] Pseudo-element is used for a blue line below the active link
- [x] Code follows all the [Code Style Rules ❗️](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)

**Pull request closed without merging**